### PR TITLE
docs: improve docstrings for some drivers

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

**Review by Yongxuanzhang — COMMENTED** [review]
Some typos. I used Grammarly to catch them ;)

File: jina/drivers/control.py [hunk 1]
```diff
@@ -66,17 +67,18 @@ def __call__(self, *args, **kwargs):
...
+      that the load is balanced between parallel `:class:`BasePea` if the scheduling `:class:`SchedulerType` is LOAD_BALANCE.
+      
+    .. note::
+        - The dealer never receives a control request from the router,
+        everytime it finishes a job and send via out_sock, it returns the envelope with control
+        request idle back to the router. The dealer also sends control request idle to the router
+        when it first starts.
+
+        - The router receives request from both dealer and upstream pusher.
+         if it is a upstream request, use LB to schedule the receiver,
```
[the remark this answers is not in the document]
Comment (JoanFM) [reply jina/drivers/control.py:80]: may u add this as suggestion so that I can automatically commit them?

File: jina/drivers/control.py [hunk 2]
```diff
@@ -66,17 +67,18 @@ def __call__(self, *args, **kwargs):
...
-    - The router receives request from both dealer and upstream pusher.
-      if it is a upstream request, use LB to schedule the receiver, mark it in the envelope
-      if it is a control request in
-
+    """Ensures that data requests are forwarded to the downstream `:class:`BasePea` ensuring
+      that the load is balanced between parallel `:class:`BasePea` if the scheduling `:class:`SchedulerType` is LOAD_BALANCE.
+      
+    .. note::
+        - The dealer never receives a control request from the router,
+        everytime it finishes a job and send via out_sock, it returns the envelope with control
```
Comment (Yongxuanzhang) [inline jina/drivers/control.py:75]:
```suggestion
        every time it finishes a job and sends via out_sock, it returns the envelope with control
```

File: jina/drivers/control.py [hunk 3]
```diff
@@ -66,17 +67,18 @@ def __call__(self, *args, **kwargs):
...
+        request idle back to the router. The dealer also sends control request idle to the router
+        when it first starts.
+
+        - The router receives request from both dealer and upstream pusher.
```
Comment (Yongxuanzhang) [inline jina/drivers/control.py:79]:
```suggestion
        - The router receives requests from both dealer and upstream pusher.
```

File: jina/drivers/control.py (hunk 1 above)
Comment (Yongxuanzhang) [inline jina/drivers/control.py:80]:
```suggestion
         if it is an upstream request, use LB to schedule the receiver,
```

[the remark this answers is not in the document]
Comment (Yongxuanzhang) [reply jina/drivers/control.py:80]: Sure!

**Comment by JoanFM** [comment]
@Yongxuanzhang could you review again?

**Review by Yongxuanzhang — APPROVED** [review]
Some args, kwards are missing. But I think it's ok to leave them? Others LGTM. 👍

**Pull request merged**